parlai/agents/rag/dpr.py

```diff
@@ -203,13 +203,19 @@ def _get_build_options(cls, opt: Opt):
         try:
             # determine if loading a RAG model
             loaded_opt = Opt.load(f"{query_path}.opt")
-            if loaded_opt['model'] == 'rag' and loaded_opt['query_model'] in [
+            document_path = loaded_opt.get('dpr_model_file', document_path)
```

Comment:
Does this slow things down if you have a `fid` model and you load `document_path` later again? In that case does it make sense to move this to the else condition of the following if?

Reply:
im not entirely sure what you're asking here --> we're only getting the document path from the `.opt` file. We should *never* actually build a FiD model in this scenario